<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LisaLaskudeAkenClass</class>
 <widget class="QDialog" name="LisaLaskudeAkenClass">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>237</width>
    <height>273</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lisalasud</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" rowspan="5" colspan="2">
    <widget class="QListWidget" name="lisaLasud"/>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="lisaNupp">
     <property name="text">
      <string>Lisa</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="eemaldaNupp">
     <property name="text">
      <string>Eemalda</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="ulesNupp">
     <property name="text">
      <string>Ãœles</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="allaNupp">
     <property name="text">
      <string>Alla</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>87</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>34</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="1">
    <widget class="QPushButton" name="olguNupp">
     <property name="text">
      <string>Olgu</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QPushButton" name="loobuNupp">
     <property name="text">
      <string>Loobu</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>loobuNupp</sender>
   <signal>clicked()</signal>
   <receiver>LisaLaskudeAkenClass</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>334</x>
     <y>277</y>
    </hint>
    <hint type="destinationlabel">
     <x>341</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>olguNupp</sender>
   <signal>clicked()</signal>
   <receiver>LisaLaskudeAkenClass</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>261</x>
     <y>272</y>
    </hint>
    <hint type="destinationlabel">
     <x>183</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
